main.go

```diff
@@ -1,11 +1,12 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"net/http"
 	"os"
 	"os/signal"
 	"syscall"
-
 	"github.com/bwmarrin/discordgo"
```

Comment:
Other dependencies are usually separated for a reason. I think better to not remove this.

Reply:
oh thanks.. seems like space is very important in go :) will put it back